src/main/java/evelyn/task/Deadline.java

```diff
@@ -0,0 +1,33 @@
+package evelyn.task;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a Deadline task.
+ */
+public class Deadline extends Task{
+    private LocalDate date = null;
+    private String time = null;
+
+    /**
+     * Constructor of a Deadline task.
+     * @param desccription Description of a Deadline task.
+     * @param deadline Deadline of a Deadline task.
+     * @param isMarked Status of a Deadline task.
+     */
+    public Deadline(String desccription, String deadline, boolean isMarked) {
```

Comment:
Perhaps there has been a mistype here and the following line, where you may have entered "desccription" instead of "description".
```suggestion
    public Deadline(String description, String deadline, boolean isMarked) {
```